mm2src/coins/z_coin.rs

```diff
@@ -756,11 +759,11 @@ impl AsRef<UtxoCoinFields> for ZCoin {
 }
 
 #[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "lowercase")]
 pub enum SyncStartPoint {
-    #[serde(rename = "date")]
     Date(u64),
-    #[serde(rename = "height")]
     Height(u64),
+    Earliest,
```

Comment:
Was this requested by the GUI team? I understand that they requested that we detect the earliest if the date or height provided is pre-sapling.

Reply:
Yes, this will be used for `full sync` (from the earliest block aka sapling activation height)